docs/cli.md

```diff
@@ -340,6 +341,7 @@ done
 | `database hostname` | Get the MongoDB Atlas Data Federation instance hostname and connection URI. See [Configure data query](/use-cases/sensor-data-query-with-third-party-tools/#configure-data-query) | - |
 | `dataset add` | Add a new image to an existing dataset by its file id, or add a group of images by specifying a filter | `filter` |
 | `dataset remove` | Remove an existing image from a dataset by its file id, or remove a group of images by specifying a filter | `filter` |
+| `dataset export` | Download all the data from a dataset | `datasetFlagDatasetID`: Dataset ID of the dataset to download. To retrieve these IDs, navigate to your dataset’s page in the [Viam app](https://app.viam.com), click **…** in the left-hand menu, and click **Copy dataset ID**.</li><li>`datasetFlagIncludeJSONLines`: Set to `true` to include JSON Lines files for local testing. _(optional)_</li><li>`dataFlagParallelDownloads`: Number of download requests to make in parallel. _(optional)_ Default: `100`</li></ul> |
```

Comment:
And now that I look at this again, these are named arguments that should go in the named arguments table below. not in this table.

Reply:
yup sounds good will fix, thanks naomi